Integrations/src/test/java/io/deephaven/integrations/learn/ComputerTest.java

```diff
@@ -0,0 +1,126 @@
+package io.deephaven.integrations.learn;
+
+import io.deephaven.db.v2.InMemoryTable;
+import io.deephaven.db.v2.sources.ColumnSource;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+public class ComputerTest {
+
+    private static InMemoryTable table;
+    private static Input[] inputs;
+
+    @BeforeClass
+    public static void createTable() {
+        table = new InMemoryTable(
+                new String[]{"Column1", "Column2", "Column3"},
+                new Object[]{
+                        new int[]{1, 2, 1, 2, 3, 1, 2, 3, 4},
+                        new long[]{2L, 4L, 2L, 4L, 6L, 2L, 4L, 6L, 8L},
+                        new double[]{5.1, 2.8, 5.7, 2.4, 7.5, 2.2, 6.4, 2.1, 7.8}
+                });
+        inputs = new Input[]{new Input(new String[]{"Column1","Column2"}, args -> args), new Input("Column3", args -> args)};
```

Comment:
I think this goes away.

Reply:
If you are going to use this, it should be calling the function below.  It is convenient to have it here, but you create problems if you forget to override it in the cases where that is necessary.